postfix/files/mapping.j2

```diff
@@ -1,11 +1,14 @@
 # Managed by config management
 
+{%- if colon is not defined %}
```

Comment:
While the purpose of this variable appears clear in the context of the diff, it is less obvious when reading the template alone. This example should be a good start
```
{# Files like mail alias database managed by this formula require colon to separate data #}
```